lib/tokenize.cpp

```diff
@@ -8463,6 +8463,16 @@ void Tokenizer::findGarbageCode() const
     // Operators without operands..
     const Token *templateEndToken = nullptr;
     for (const Token *tok = tokens(); tok; tok = tok->next()) {
+        // Skip preprocessor directives
+        if(Token::Match(tok, "# line|pragma")) {
+            if(!Token::Match(tok->next(), "if|elif|else|endif|defined|ifdef|ifndef|define|undef|include|line|error|pragma"))
```

Comment:
I don't see how next token could be "if".

Reply:
Oops that is a mistake.